content/blog/2024/07/kubewarden-1-15-release.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Kubewarden 1.15 release
+authors:
+  - Jose Guilherme Vanz
+date: 2024-07-17
+---
+
+# Kubewarden v1.15.0 release
+
+We are thrilled to announce the release of Kubewarden v1.15.0! This version
+comes packed with CEL policy updates, controller enhancements, and fixes that
+make Kubewarden even more robust and user-friendly.
+
+## Enhanced PolicyServer CRD with Tolerations 
+
+One of the standout features of Kubewarden v1.15 is the extension of the
+PolicyServer Custom Resource Definition (CRD) to include a list of Toleration
+objects to be used in the deployment created for the PolicyServer. This
+enhancement allows users to specify tolerations directly within the
+PolicyServer CRD, simplifying the management of taints and tolerations across
+your Kubernetes cluster. By integrating this directly into the CRD, Kubewarden
+ensures a more streamlined and consistent approach to handling node
+tolerations, making it easier to deploy policies in diverse and dynamic
+environments.
+
+## Centralized Management of Tolerations, Affinity, and Anti-Affinity 
+
+In line with making the deployment process more efficient, Kubewarden v1.15
+introduces a central place where tolerations, affinity, and anti-affinity rules
+are specified and enforced across all components of the stack (controller deployment,
+audit scanner cronjob and default PolicyServer). This centralized management
+approach not only simplifies the configuration process but also ensures
+consistency and reduces the chances of configuration drift. By defining these
+rules once and applying them universally, users can achieve a more reliable and
+predictable policy enforcement setup.
+
+:::warning
+The affinity configuration previously present in the default policy server values 
+has been removed. Then, if you were using it, you need to migrate your configuration
+:::
+
+## Technical Debt Reduction in Kubewarden Controller 
+
+In v1.15, significant efforts have been made to reduce technical debt in the
+Kubewarden controller, ensuring better performance, easier maintenance, and a
+more robust foundation. Key changes include:
+
+- **Updated Golang Linter**: The stricter linter catches potential issues
+  early, ensuring higher code quality.
+- **Removed --default-policy-server CLI Flag**: The default policy server name
+  is now always "default," simplifying configuration.
+- **Updated to Kubebuilder 4.0.0**: The controller code now uses Kubebuilder
+  4.0.0.
+- **Refactored Reconciler Code**: The code is split into multiple
+  subreconcilers following the Subreconciler pattern, enhancing manageability
+  and modularity.
+- **File/Package Renaming**: Files and packages have been renamed for better
+  readability and navigation.
+
+These changes improve functionality and set the stage for more streamlined
+development and maintenance, reflecting a commitment to maintaining a
+high-quality codebase.
+
```

Comment:
@fabriziosestito pointed out the old code (before 1.15) used to kubernetes clients, one of them was not using the cache feature. With the refactoring this client is gone, meaning we have further reduced the amount of requests made against the Kubernetes API :partying_face:

Reply:
@jvanz can you add some lines about this improvement? Maybe this could be worth a new section "performance improvements"

Reply:
I've added a new item for this topic